docs/modules/ROOT/pages/mgmt_services/port_manager.adoc

```diff
@@ -0,0 +1,600 @@
+. Support create and update of a list of ports.
+. Working together with other services including vpc manager, subnet manager, mac manager, security group manager and dataplane manager.
+. Support asynchronous interaction with other services
+. Rollback is supported when CRUD operation occurs exception.
+. Support query operation with any field of port as parameters.
+
+
+== REST APIs
+
+=== API Snapshot
+
+[width="100%",cols="22%,12%,50%,17%"]
+|===
+|*API Name* |*Method* |*Request*|*Response*
+
+|Create port
+|POST
+|/project/{project_id}/ports
+|port state
+<<Create_Port,[sample]>>
+
+|Create port bulk
+|POST
+|/project/{project_id}/ports/bulk
+|port state list
+<<Create_Port_Bulk,[sample]>>
+
+|Update port
+|PUT
+|/project/{project_id}/ports/{port_id}
+|port state
+<<Update_Port,[sample]>>
+
+|Delete port
+|POST
+|/project/{project_id}/ports/{port_id}
+|None
+<<Delete_Port,[sample]>>
+
+|Get port
+|GET
+|/project/{project_id}/ports/{port_id}
+|port state
+<<Get_Port,[sample]>>
+
+|List port
+|GET
+|/project/{project_id}/ports
+|port state list
+<<List_Port,[sample]>>
+|===
+
+=== API Specification
+
+anchor:Create_Port[]
+**(1) Create port**
+
+* Method: `POST`
+
+* Request: `/project/{project_id}/ports`
+
+* Request Parameter: `@PathVariable String projectId`
+
+* Response: port state
+* Normal response codes: 200
+* Error response codes: 400, 412, 500
+
+* Example
+
+....
+Request:
+http://127.0.0.1:8080/project/3d53801c-32ce-4e97-9572-bb966f4de79c/ports
+
+Body:
+{
+	"port": {
+		"id": "3d53801c-32ce-4e97-9572-bb966f4aa53e",
+		"vpc_id": "3d53801c-32ce-4e97-9572-bb966f4d175e",
+		"tenant_id": "3d53801c-32ce-4e97-9572-bb966f476ec",
+		"fixed_ips": [{
+			"subnet_id": "3d53801c-32ce-4e97-9572-bb966f4056b",
+			"ip_address": "11.11.11.100"
+		}],
+		"security_groups": ["3d53801c-32ce-4e97-9572-bb966f4d45ca"],
+		"allowed_address_pairs": [{
+			"ip_address": "11.11.11.101",
+			"mac_address": "00:01:6C:06:A6:29"
+		}]
+	}
+}
+
+Response:
+{
+	"port": {
+		"fastPath": false,
+		"project_id": "3d53801c-32ce-4e97-9572-bb966f4de79c",
+		"id": "3d53801c-32ce-4e97-9572-bb966f4aa53e",
+		"name": null,
+		"description": null,
+		"vpc_id": "3d53801c-32ce-4e97-9572-bb966f4d175e",
+		"tenant_id": "3d53801c-32ce-4e97-9572-bb966f476ec",
+		"admin_state_up": false,
+		"mac_address": "00:01:6C:06:A6:29",
+		"veth_name": null,
+		"fast_path": false,
+		"device_id": null,
+		"device_owner": null,
+		"status": null,
+		"fixed_ips": [{
+			"subnet_id": "3d53801c-32ce-4e97-9572-bb966f4056b",
+			"ip_address": "11.11.11.100"
+		}],
+		"allowed_address_pairs": [{
+			"ip_address": "11.11.11.101",
+			"mac_address": "00:01:6C:06:A6:29"
+		}],
+		"extra_dhcp_opts": null,
+		"security_groups": ["3d53801c-32ce-4e97-9572-bb966f4d45ca"],
+		"binding:host_id": null,
+		"binding:profile": null,
+		"binding:vif_details": null,
+		"binding:vif_type": null,
+		"binding:vnic_type": null,
+		"network_ns": null,
+		"dns_name": null,
+		"dns_assignment": null
+	}
+}
```

Comment:
We don't have created_at and updated_at value. Refer to latest VPC/Subnet manager implementation, which just added those fields.